.github/workflows/ci.yml

```diff
@@ -15,13 +15,18 @@ jobs:
           - gemfiles/rails_6_1.gemfile
           - gemfiles/rails_7_0.gemfile
           - gemfiles/rails_7_1.gemfile
+          - gemfiles/rails_7_2.gemfile
         exclude:
           - ruby: '3.2'
             gemfile: gemfiles/rails_5_2.gemfile
           - ruby: '3.1'
             gemfile: gemfiles/rails_5_2.gemfile
+          - ruby: '3.0'
+            gemfile: gemfiles/rails_7_2.gemfile
           - ruby: '3.0'
             gemfile: gemfiles/rails_5_2.gemfile
+          - ruby: '2.7'
+            gemfile: gemfiles/rails_7_2.gemfile
```

Comment:
Rails 7.2 has a minimum version of Ruby 3.1. Not sure if I should add an exclusion for Ruby 2.6. Got a sense from this PR that we may be removing Ruby 2.6 support? https://github.com/Betterment/delayed/pull/42

Reply:
Yeah, this is fine for now! Fully dropping Ruby 2.6 doesn't need to be part of this PR.